cloudify_awssdk/ecs/resources/service.py

```diff
@@ -34,9 +34,10 @@
 
 RESOURCE_TYPE = 'ECS Service'
 CLUSTER = 'cluster'
-CLUSTER_TYPE = 'cloudify.nodes.aws.ECS.Cluster'
+CLUSTER_TYPE = 'cloudify.nodes.aws.ecs.Cluster'
```

Comment:
Do we need to support both cases for backward compatibility?

Reply:
This is just exists on one repo so I just rename them to match what we already have on the repo